sdcm/cluster_k8s/__init__.py

```diff
@@ -2533,8 +2533,8 @@ def prefill_cluster(self, dataset_name: str):
             if create_schema:
                 create_schema = False
 
-        result = SstableLoadUtils.validate_data_count_after_upload(node=node)
-        assert int(result) == test_data[0].keys_num, "Data was not inserted"
+            result = SstableLoadUtils.validate_data_count_after_upload(node=node)
```

Comment:
It not correct. We do not need to perform this validation on every node.

Maybe I could name `node` in the `for` loop as `current_nod`, so it will be clear:

```
    -        for node in self.nodes:
    +       for current_node in self.nodes:
            
            
```

Reply:
The rename is not needed, it doesn't help.

Reply:
> The rename is not needed, it doesn't help.

It add mess, because of `node` variable is created before the loop:

```
node = self.nodes[0]
```
This is the node that is used for validation.
Changing the "node" inside `for` loop prevent this mess